modules/weko-search-ui/weko_search_ui/admin.py

```diff
@@ -500,6 +506,67 @@ def export_template(self):
                    else urlencode({'filename': file_name}))
             })
 
+class ItemBulkExport(BaseView):
+    """BaseView for Admin Export."""
+
+    @expose('/', methods=['GET'])
+    def index(self):
+        """Renders admin bulk export page.
+
+        :param
+        :return: The rendered template.
+        """
+        return self.render(
+            WEKO_SEARCH_UI_ADMIN_EXPORT_TEMPLATE
+        )
+
+    @expose('/export_all', methods=['GET'])
+    def export_all(self):
```

Comment:
need check export all is running or not then execute export progress because this route can call from url without UI